{{cookiecutter.app_name}}/bootstrap/database.go

```diff
@@ -0,0 +1,24 @@
+package bootstrap
+
+import (
+	"log"
+	"strings"
+
+	"github.com/nimblehq/{{cookiecutter.app_name}}/helpers"
+
+	"github.com/gin-gonic/gin"
+	"github.com/spf13/viper"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+// Init database and set to viper config
```

Comment:
This kind of comment is kind of redundant as it repeats the method name. Prefer to avoid.

Reply:
Removed in 872e92a42b0e2c6af75ac85fec495328f887bcbc